api/v1beta1/modelregistry_types.go

```diff
@@ -306,10 +307,9 @@ type ModelRegistrySpec struct {
 	// Configuration for REST endpoint
 	Rest RestSpec `json:"rest"`
 
-	//+kubebuilder:required
-
-	// Configuration for gRPC endpoint
-	Grpc GrpcSpec `json:"grpc"`
+	// Deprecated: Configuration for gRPC endpoint is deprecated and will be removed in a future release
+	//+optional
+	Grpc *GrpcSpec `json:"grpc,omitempty"`
```

Comment:
Is this required, or can we avoid changing the API? I'd recommend introducing a new API version instead to cleanup all deprecated properties in the future.